src/pages/developing/svelte-tutorial/step-1.mdx

```diff
@@ -0,0 +1,486 @@
+Carbon Design System white theme. Because it is imported in `__layout.svelte`,
+the styles will be applied to all pages of the app.
+
+In a later step, we will optimize the CSS when building the app for production.
+
+We will also need to add a `slot` to the markup in `__layout.svelte`. This will
+allow content from other pages to be rendered with this default layout.
+
+```html path=src/routes/__layout.svelte
+<script>
+  import 'carbon-components-svelte/css/white.css';
+</script>
+
+<slot />
+```
+
+Next, we'll import a `Button` from Carbon to test that our dependencies are
+working properly. At the top of `src/routes/index.svelte`, import the `Button`
+by adding the following:
+
+```html path=src/routes/index.svelte
+<script>
+  import { Button } from 'carbon-components-svelte';
+</script>
+```
+
+In `src/routes/index.svelte`, you can now replace:
+
+<!-- prettier-ignore-start -->
+```html path=src/routes/index.svelte
+<div>
+  Hello Carbon! Well, not quite yet. This is the starting point for the Carbon tutorial.
+</div>
+```
+<!-- prettier-ignore-end -->
+
+with:
+
+<!-- prettier-ignore-start -->
+```html path=src/routes/index.svelte
+<Button>Button</Button>
+```
+<!-- prettier-ignore-end -->
+
+Congratulations, you've imported your first component! You should see a Carbon
+styled button on the page.
+
+## Add UI Shell
+
+Next, we're going to create a Svelte component called `TutorialHeader` to use
+with the UI Shell Carbon component. In the `src` directory, create a `lib`
+directory and inside of that, a `TutorialHeader.svelte` file.
+
+```bash
+src/lib/
+└──TutorialHeader.svelte
+```
+
+Next we'll import our Carbon UI Shell components into `TutorialHeader.svelte`.
+Set up the file like so:
+
+<!-- prettier-ignore-start -->
+```html path=src/lib/TutorialHeader.svelte
+<script>
+  import {
+    SkipToContent,
+    Header,
+    HeaderNav,
+    HeaderNavItem,
+    HeaderUtilities,
+    HeaderGlobalAction,
+  } from 'carbon-components-svelte';
+  import { page } from '$app/stores';
+</script>
+
+<Header company="IBM" platformName="Carbon Tutorial" href="/">
+  <div slot="skip-to-content">
+    <SkipToContent />
+  </div>
+  <HeaderNav>
+    <HeaderNavItem
+      text="Repositories"
+      href="/repos"
+      sveltekit:prefetch
+      aria-current="{$page.path === '/repos' ? 'page' : undefined}"
+    />
+  </HeaderNav>
+  <HeaderUtilities> </HeaderUtilities>
+</Header>
+```
+<!-- prettier-ignore-end -->
+
+<InlineNotification>
+
+**Note:** When creating navigation headers, it's important to have a
+`Skip to content` link so keyboard users can skip the navigation items and go
+straight to the main content.
+
+</InlineNotification>
+
+### Import icons
+
+Now let's import the icons. In the `TutorialHeader.svelte` script block, we need
+to import each individual icon we will use.
+
+```javascript path=src/lib/TutorialHeader.svelte
+import {
+  Notification20,
+  UserAvatar20,
+  AppSwitcher20,
+} from 'carbon-icons-svelte';
+```
```

Comment:
are we supposed to already have `carbon-icons-svelte` installed at this point? tutorial doesn't mention it 
<img width="951" alt="image" src="https://user-images.githubusercontent.com/40550942/200347102-0319c8fa-7160-4883-ad2b-5b94ff178af2.png">